### PR TITLE
implemented Helper for CGRect/SizeMake

### LINKED ISSUE #8: Implement Helper for CGSizeMake/CGRectMake

### PR DESCRIPTION
**Pull request by maximschlei**
closes #8

### CODE REVIEW

File: SCICoreTools/Extensions/CGRect+Extension.swift [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+public func CGRectMake(_ x: CGFloat, _ y: CGFloat, _ widht: CGFloat, _ height: CGFloat) -> CGRect {
```
Comment (jakobvogel) [inline SCICoreTools/Extensions/CGRect+Extension.swift:3]: CGMakeRect?

File: SCICoreTools/Extensions/CGSize+Extension.swift [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+public func CGSizeMake(_ width: CGFloat, _ height: CGFloat) -> CGSize {
```
Comment (jakobvogel) [inline SCICoreTools/Extensions/CGSize+Extension.swift:3]: CGMakeSize?

**Review by jakobvogel — COMMENTED** [review]
For historical and grammatical reasons, I would flip verb and noun.

File: SCICoreTools/Extensions/CGSize+Extension.swift (hunk 2 above)
Comment (maximschlei) [reply SCICoreTools/Extensions/CGSize+Extension.swift:3]: Oldschool func name before swift 3

File: SCICoreTools/Extensions/CGRect+Extension.swift (hunk 1 above)
Comment (maximschlei) [reply SCICoreTools/Extensions/CGRect+Extension.swift:3]: Oldschool func name before swift 3

**Pull request merged**